exporter/prometheusexporter/config.go

```diff
@@ -34,6 +34,9 @@ type Config struct {
 
 	// EnableOpenMetrics enables the use of the OpenMetrics encoding option for the prometheus exporter.
 	EnableOpenMetrics bool `mapstructure:"enable_open_metrics"`
+
+	// AddUnitSuffixes controls whether suffixes are added to metric names. Defaults to true.
+	AddUnitSuffixes bool `mapstructure:"add_metric_suffixes"`
```

Comment:
Can we name the go API the same as the user config option? 
```suggestion
	// AddMetricSuffixes controls whether suffixes are added to metric names. Defaults to true.
	AddMetricSuffixes bool `mapstructure:"add_metric_suffixes"`
```